docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -132,7 +132,9 @@ To deserialize from a string or a file, call the <xref:System.Text.Json.JsonSeri
 
 The following example reads JSON from a string and creates an instance of the `WeatherForecast` class shown earlier for the [serialization example](#serialization-example):
 
-[!code-csharp[](snippets/system-text-json-how-to/csharp/RoundtripToString.cs?name=SnippetDeserialize)]
+```csharp
+WeatherForecast weatherForecast = JsonSerializer.Deserialize<WeatherForecast>(jsonString);
+```
```

Comment:
I updated this because the doc read like this, which was confusing to me:

![image](https://user-images.githubusercontent.com/24882762/95642054-ef1e3200-0a5a-11eb-9cba-179462affad4.png)

So there's a mismatch between the class name, WeatherForecast, and the type in the generic type parameter (WeatherForecastWithPOCOs).

Reply:
The "class shown earlier for the serialization example" is WeatherForecastWithPOCOs, so it might be better here to change the intro sentence rather than the code -- "...creates an instance of the `WeatherForecastWithPOCOs` class shown earlier..."